### ISSUE TITLE
BUG: Error occurs when creating a second event type

### ISSUE THREAD

**Issue by fangyu3** [hatchways/team-beluga#62]
There is no issue creating the first event. However, error occurs when trying to create second event.

From the output of terminal it seems the user_id is missing when creating the second event. 

sqlalchemy.exc.DataError: (psycopg2.errors.InvalidTextRepresentation) invalid input syntax for type integer: ""
LINE 1: ...es (user_id, title, url, duration, color) VALUES ('', 'asd',...
                                                             ^

[SQL: INSERT INTO event_types (user_id, title, url, duration, color) VALUES (%(user_id)s, %(title)s, %(url)s, %(duration)s, %(color)s) RETURNING event_types.id]
[parameters: {'user_id': '', 'title': 'asd', 'url': 'asd', 'duration': 30, 'color': '#3E200B'}]
(Background on this error at: http://sqlalche.me/e/14/9h9h)

**Comment by fangyu3** [comment]
This might have something to do with usercontext. When the page refreshes, user id is gone. 

Should be solved once the usercontext/private route branches are merged. Will double check again

**Comment by JoshL579** [comment]
Yeah, this is the problem I was trying to solve in PrivateRoute, this is all about `Async`. I believe this problem will disappear once I merge the PrivateRoute
